recipes/pylibjpeg-libjpeg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pylibjpeg-libjpeg" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pylibjpeg_libjpeg-{{ version }}.tar.gz
+  sha256: ffb736acee1dfd446f387d45dfec83a144c7971e5cde290d9aafefa5dee99dd0
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [win]
+  ignore_run_exports:
+    - cython
+    - numpy
+    - python
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - cython >=3.0.10
+    - numpy >=2.0.0,<3.0.0
+    - pip
+    - poetry-core >=1.8,<2
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=2.0.0,<3.0.0
+    - cython >=3.0.10
+    - libcxx 
+    - libgcc
```

Comment:
```suggestion
```
These should not be needed as run time dependencies as they are automatically added by using `{{ compiler('cxx') }}`